src/model/index.py

```diff
@@ -29,28 +29,35 @@ def serialize(self):
 
     def get_primary_keys(self, value_name):
         Index.__validate_value_name__(value_name)
+        primary_keys = []
         path = os.path.join(self.__path__, "{}.json".format(value_name))
         if os.path.exists(path):
             with open(path, 'r') as file:
-                return json.load(file)
-        else:
-            return []
+                primary_keys = json.load(file)
+        return primary_keys
 
     @staticmethod
     def __update_value__(path, value_name, primary_keys):
         with open(os.path.join(path, "{}.json".format(value_name)), 'w') as file:
             json.dump(primary_keys, file)
 
     def add_value(self, value_name, primary_key):
-            value = self.get_primary_keys(value_name)
-            value.append(primary_key)
-            self.__update_value__(self.path, value_name, value)
+        value = self.get_primary_keys(value_name)
+        value.append(primary_key)
+        self.__update_value__(self.__path__, value_name, value)
 
     def remove_value(self, value_name, primary_key):
         value = self.get_primary_keys(value_name)
         if primary_key in value:
             value.remove(primary_key)
         if not value:
-            pathlib.Path(os.path.join(self.path, "{}.json".format(value_name))).unlink()
+            pathlib.Path(os.path.join(self.__path__, "{}.json".format(value_name))).unlink()
         else:
-            self.__update_value__(self.path, value_name, value)
+            self.__update_value__(self.__path__, value_name, value)
```

Comment:
Why should I care about the file here to delete it or not. `self. __update_value__ ` should take care about this